docs/VehicleInfo/SubscribeVehicleData/index.md

```diff
@@ -44,6 +44,7 @@ Purpose
 |engineOilLife|Boolean|false||
 |electronicParkBrakeStatus|Boolean|false||
 |cloudAppVehicleID|Boolean|false||
+|handsOffSteering|Bolean|false||
```

Comment:
```suggestion
|handsOffSteering|Boolean|false||
```

Reply:
fixed in aa1b73342f4e28b176864bd248d88635519290bc